### ISSUE TITLE
Distinction among Serenity Web Test (Selenium) and Serenity Non-Web Test

### ISSUE THREAD

**Issue by ereticul** [serenity-bdd/serenity-core#41]
Hi John, 

Besides of web tests (selenium), we can just create and NON-web tests (back-end, integration unit tests), and still have this wonderful Serenity Reporting capability.

But I noticed in the Serenity Report at the test outcome level that it's displayed the "Firefox" image, however no instance of browser has been opened.

Yes, it seems to be a cosmetic/ trivial issue that the image is displayed; So far now the default driver is set to "firefox", so due to that the image is displayed I believe. This kind of false positive output persuades to the wrong impression. 

Would it be possible to hide this image for non-web tests?

- Andy.

**Comment by ereticul** [comment]
John, 

What do you think about something like that?

```javascript
public void testStarted(final String testMethod) {
        TestOutcome newTestOutcome = TestOutcome.forTestInStory(testMethod, testSuite, testedStory);
        if (driver != null) {
 # New line with this check added:
 if(webdriverManager.getActiveWebdriverCount() > 0) {
                newTestOutcome.setDriver(webdriverManager.getCurrentDriverName());
            }
        }
```
....
and we've already has a check in default.ftl, if above won't be set then the image won't be displayed.
```javascript
<#if (testOutcome.driver)??>
<span style="float:right"><img src="images/driver-${testOutcome.driver}.png" height="20" alt="${testOutcome.driver}" title="${testOutcome.driver}" /></span>
</#if>
```

I think it can work, please sorry I'm not able to check that, the Serenity-core environment is not set up at my workstation. 

- Andy

**Comment by wakaleo** [comment]
Fixed in Serenity 1.0.44
